impl/sql/mutationstorage/queue_test.go

```diff
@@ -76,6 +77,39 @@ func TestRandLog(t *testing.T) {
 	}
 }
 
+func BenchmarkSend(b *testing.B) {
+	ctx := context.Background()
+	logID := int64(1)
+	m := newForTest(ctx, b, logID)
+
+	update := &pb.EntryUpdate{Mutation: &pb.SignedEntry{Entry: []byte("xxxxxxxxxxxxxxxxxx")}}
+	for _, tc := range []struct {
+		batch int
+	}{
+		{batch: 1},
+		{batch: 2},
+		{batch: 4},
+		{batch: 8},
+		{batch: 16},
+		{batch: 32},
+		{batch: 64},
+		{batch: 128},
+		{batch: 256},
+	} {
+		b.Run(fmt.Sprintf("%d", tc.batch), func(b *testing.B) {
+			updates := make([]*pb.EntryUpdate, 0, tc.batch)
+			for range updates {
```

Comment:
It doesn't look like this loops works as you intend: https://play.golang.org/p/PT6XW2ITR7j
I think this benchmark ends up measuring the number of empty `Send` calls per second.